client/index.html

```diff
@@ -2,10 +2,16 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <link rel="stylesheet" href="./index.css">
-  <script type="module" src="./index.js" defer></script>
   <meta name="color-scheme" content="dark light">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="A page to display compatible browsers from a browserslist string">
+  <!-- TODO Add og.png (1200x630) <meta property="og:image" content="/assets/og.png"> -->
+  <link rel="icon" href="/assets/favicon.ico" sizes="any">
```

Comment:
Can we put it to the root to have `/favicon.ico`?

Reply:
Good.

 I decided to move everything icons to `/public` folder because `.webmanifest` not parsed in simple static server from `/dist` folder ([as here](https://github.com/evilmartians/oklch-picker/tree/main/public)). 


Maybe something will change here https://github.com/browserslist/browserl.ist/pull/319#discussion_r939744554

Reply:
**Problem**
![image](https://user-images.githubusercontent.com/22644149/183532494-4e3b5568-7343-4a37-9898-55da8f27f38b.png)

- Somehow Detect files from the Vite's `/public` folder and give it 1 hour of cache control (ugly)
- Create a subfolder "public/icons" and allocate cache 3600 for it (simple)
- Try to make sure that Vite does not add manifest as data:base64... and changed the paths in it to `icon-192.png` and `icon-512.png` when `build` (difficult?)
- Maybe we can not worry and add `?cache123` flag if we need to replace the icon (funny)

Reply:
**Problem**
![image](https://user-images.githubusercontent.com/22644149/183532494-4e3b5568-7343-4a37-9898-55da8f27f38b.png)

- Somehow Detect files from the Vite's `/public` folder and give it 1 hour of cache control (simple, not obviusly)
- Create a subfolder `public/icons` and allocate cache 3600 for it (simple)
- Try to make that Vite's `build` script does not add manifest as `href="data:base64...` and changed the paths in it to `icon-192.png` and `icon-512.png` (difficult?)
- Maybe we can not worry and add `?cache123` flag if we need to replace the icon (dirty, old-school)

Reply:
The same practice as we have for `/index.html`:

* In Node.js we can use infinite in-memory cache for any static files, since new version will be available only after deploy.
* We will still need a 5-15 minutes cache in HTTP headers for non-cache-buster files

`data:base64` and `?cache123` is not an option for `/index.html` and `/faicon.ico`. Some tools may load favicons just by `host + "/faicon.ico"`

Reply:
> We will still need a 5-15 minutes cache in HTTP headers for non-cache-buster files

Great. It remains to solve this problem in https://github.com/browserslist/browserl.ist/pull/319

We need to understand what file with a cache-buster and what is not. Maybe regexp
`${fileName}.(check 8 random symbols).${fileExt}`

Reply:
This is what I used on some project :D Works good